app/views/form/_date_question.html.erb

```diff
@@ -3,7 +3,7 @@
 <%= f.govuk_date_field question.id.to_sym,
   caption: caption(caption_text, page_header, conditional),
   legend: legend(question, page_header, conditional),
-  hint: { text: question.hint_text&.html_safe || "For example, 1 9 2022." },
+  hint: { text: question.hint_text&.html_safe || "For example, #{example_date_in_tax_year_of(lettings_log.startdate).to_formatted_s(:govuk_date_number_month)}" },
```

Comment:
```suggestion
  hint: { text: question.hint_text&.html_safe || "For example, #{example_date_in_tax_year_of(lettings_log.startdate).to_formatted_s(:govuk_date_number_month)}" },
```

You've made a helper to do this, which is great, but you can help yourself even more by formatting the date within it :)

Reply:
yes I have moved formatting into method in collecitontimehelper now.